tools/performance/engine-benchmarks/README.md

```diff
@@ -26,3 +26,19 @@ results.
 
 One can also download only a CSV file representing all the selected benchmark
 results with `bench_download.py --create-csv`.
+
+## Contribute
+Run local tests with:
+```bash
+python -m unittest --verbose bench_tool/test*.py
+```
+
+## Relation to GH Actions
+The `bench_download.py` script is used in [Benchmarks Upload](https://github.com/enso-org/enso/actions/workflows/bench-upload.yml)
```

Comment:
Definitely not for now, but I'm wondering we could at some point move this script into Enso :)